server/server.js

```diff
@@ -35,14 +36,32 @@ app.use( (req,res,next) => {
 app.set("views", "./views");
 app.set("view engine", "ejs");
 
+mongoose.connect('mongodb://localhost:27017/TESTLTI', {
+  useNewUrlParser: true},
+  (err) => {
+    if(err) {
+      return console.log(err);
+    }
+  }
+);
+mongoose.Promise = Promise;
+
+  registerPlatform(
+    'https://demo.moodle.net',
+    'Grading Tool',
```

Comment:
Wouldn't the consumerName be Demo's Moodle or SDCS's Moodle or Our LMS or something like that?